filesystem/azure.go

```diff
@@ -212,7 +212,7 @@ func (f *AzureFile) Write(b []byte) (int, error) {
 		return 0, err
 	}
 	if err == io.EOF {
-		return 0, fmt.Errorf("Need blob content")
+		return 0, fmt.Errorf("Need write content")
```

Comment:
> But the old code is correct to achieve this check function, when first read it, if has something, it will not return io.EOF(only read nothing, it return io.EOF).

Now you understand what I have suggested.
But let me ask this question again: can you explain why the first chunk is a special case (need it to be full)?